### PR TITLE
Fixes issue #34 and #36:Added option for adding emoji icon and it also fixes issue #34

### PR DESCRIPTION
**Pull request by amitiwary999**
fixes issue #34 and #36 Emoji icons will be displayed when user will click on image(left one) present in edittext so by this I also fixes issue 34.

### CODE REVIEW

**Comment by betterclever** [comment]
@amitiwary999 Commit all the required changed files. You have excluded the build.gradle which must have changd since you included a 3rd party library.

**Comment by amitiwary999** [comment]
My mistake ..will commit with build.gradle by today evening

**Comment by daminisatya** [comment]
@amitiwary999 Can you add a screenshot. Also please squash your commits.

**Comment by shivenmian** [comment]
Build failing.

File: build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```
Comment (betterclever) [inline build.gradle]: An older version is being used here. Use the latest version. Android Studio 2.2 as being used by us uses the version 2.2.0.

File: app/build.gradle [hunk 2]
```diff
@@ -42,4 +42,7 @@ dependencies {
     compile 'com.github.bumptech.glide:glide:3.7.0'
     compile "com.android.support:support-v4:$support_lib_version"
 
+    compile 'com.vanniktech:emoji:0.3.0'
+    compile 'com.vanniktech:emoji:0.4.0-SNAPSHOT'
```
Comment (betterclever) [inline app/build.gradle]: Please check whether both versions are required, as far as I think, SNAPSHOT version alone should do.

File: app/src/main/java/org/fossasia/susi/ai/MainActivity.java [hunk 3]
```diff
@@ -33,16 +45,20 @@
...
     @BindView(R.id.coordinator_layout)
     CoordinatorLayout coordinatorLayout;
     @BindView(R.id.rv_chat_feed)
     RecyclerView rvChatFeed;
     @BindView(R.id.iv_image)
     ImageView ivImage;
-    @BindView(R.id.et_message)
-    EditText etMessage;
+    @BindView(R.id.rl)
+    ViewGroup rootView;
```
Comment (betterclever) [inline app/src/main/java/org/fossasia/susi/ai/MainActivity.java]: What is r1 here? If you have changed any layout xml files according to this, add them in your next PR if you're going to open one.

**Comment by betterclever** [comment]
@amitiwary999 Are you going to open a new PR for it or should I work on the issue?

**Comment by amitiwary999** [comment]
I opened a new PR.

File: app/src/main/java/org/fossasia/susi/ai/MainActivity.java (hunk 3 above)
Comment (amitiwary999) [inline app/src/main/java/org/fossasia/susi/ai/MainActivity.java]: I recently opened a new PR with all update.

File: build.gradle (hunk 1 above)
Comment (amitiwary999) [inline build.gradle]: I tried to update my android studio but it always fail to upgrade to 2.2.0

**Pull request closed without merging**